app/graphql/types/host.rb

```diff
@@ -7,6 +7,7 @@ class Host < BaseObject
     global_id_field :id
     timestamps
     field :name, String
+    field :display_name, String, method: :to_label
```

Comment:
I suggest adding this field to the host query test https://github.com/theforeman/foreman/blob/5423d2c46a89f7f66b9d89a13f5fe4d5b481cab5/test/graphql/queries/host_query_test.rb#L3-L4